site/components/Provider/Provider.tsx

```diff
@@ -37,6 +38,7 @@ const connectors = connectorsForWallets([
       omniWallet({ chains }),
       imTokenWallet({ chains }),
       ledgerWallet({ chains }),
+      portalWallet({ chains }),
```

Comment:
Please revert these changes to the docs site. Instead, add to `packages/example` example site. This connector will be testable at a Vercel deployment once approved; this is our typical review process.